### PR TITLE
vmdk: fix build issues with gcc 10 or newer

### PR DESCRIPTION
**Pull request by dongsupark**
As gcc 10 or newer defaults to `-fno-common`, we need to define only once in a *.c file, instead of *.h that can be imported multiple times by *.c files.

Otherwise, build would fail like:
```
.../x86_64-pc-linux-gnu/bin/ld:
../build/vmdk/mkdisk.o:.../vmdk/diskinfo.h:38: multiple definition of `toolsVersion';
../build/vmdk/flat.o:.../vmdk/diskinfo.h:38: first defined here
```

Fixes https://github.com/vmware/open-vmdk/issues/12

### CODE REVIEW

**Review by keirazhang — APPROVED** [review]
Hi Dongsu,
Thanks for helping fix the issue. It looks good to me.

**Comment by dongsupark** [comment]
Thanks for the review.
Is there any chance to get this PR merged?

**Comment by keirazhang** [comment]
> Thanks for the review. Is there any chance to get this PR merged?

Hi Dongsu,
Sorry that I didn't realize you can't merge it from your side. I am contacting the repo's maintainer for merging it. It might take some time. 

Thanks

**Pull request merged**
